gravity-forms/gw-none-of-the-above-checkbox.js

```diff
@@ -21,9 +21,10 @@
  */
 $( '.gw-none-of-the-above' ).each( function() {
 
-	var $field  = $( this );
-	var $last   = $field.find( '.gchoice:last-child input' );
-	var $others = $field.find( 'input' ).not( $last );
+	var $disable_nota = false;
```

Comment:
```suggestion
	var disable_nota = false;
```

This isn't a jQuery object so we wouldn't usually prefix this with a `$`